lib/dialects/oracle/query.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+var Utils = require('../../utils')
+  , AbstractQuery = require('../abstract/query')
+  , _ = require('lodash')
+  , autocommit = true;
```

Comment:
Having a global option is not going to work - users might want some transactions with autocommit and some without

Couldn't you set `self.connection.autocommit` instead?